app/wama.web/Views/AdminConsole/UserTool/CreateNewUserAccount.cshtml

```diff
@@ -44,6 +44,15 @@
                 <label asp-for="InstitutionAffiliation"></label>
                 <select asp-for="InstitutionAffiliation" asp-items="@Html.GetEnumSelectList<InstitutionAffiliation>()" class="form-control"></select>
             </div>
+                <div class="form-group col-xs-12">
+                <label asp-for="AccountType"></label>
+                <select asp-for="AccountType" 
+                        asp-items="@Html.GetEnumSelectList<UserAccountType>()" 
+                        class="form-control" 
+                        >
+                    <option selected>@ViewBag.AccountType</option>
```

Comment:
@{
            UserAccountType activeType;

            Enum.TryParse($"{ViewData["type"]}", out activeType);

            activeType = Model?.AccountType ?? activeType;

            var accountTypes = Html.GetEnumSelectList<UserAccountType>()
                .Where(selectListItem => selectListItem.Text.Equals($"{UserAccountType.Unknown}") == false)
                .Select(selectListItem =>
                {
                    if (selectListItem.Text.Equals($"{activeType}"))
                    {
                        selectListItem.Selected = true;
                    }

                    return selectListItem;
                });
        }

        <div class="form-group col-xs-12">
            <label asp-for="AccountType"></label>
            <select asp-for="AccountType" asp-items="@accountTypes" class="form-control"></select>
        </div>